modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -15,19 +15,13 @@
 
 import ray
 from ray.util import get_node_ip_address
-from packaging import version
```

Comment:
Can we make this refactor in a separate PR? I don't want to mix these two things since they don't seem super related on the surface. I do like the change though - it definitely improves code readability!